src/common/exporter/word/exporters/outline.js

```diff
@@ -44,15 +45,23 @@ export default function exportOutline(state, namesMapping, doc, options) {
     const uniqueBeatTitleSelector = makeBeatTitleSelector(state)
     const title = uniqueBeatTitleSelector(state, beat.id)
     let paragraphs = [new Paragraph('')]
-    paragraphs.push(new Paragraph({ text: title, heading: HeadingLevel.HEADING_2 }))
```

Comment:
I think that there's a slight problem here.
In the case that we're not exporting scene cards, we probably still want this level 2 heading.
So can we add a check here like so:
```
if (!options.outline.sceneCards) {
  paragraphs.push(new Paragraph({ text: title, heading: HeadingLevel.HEADING_2 }))
}
```

Reply:
@cameronsutter, could you please clarify something about this change.

I've suggested that we now need to check whether we're not exporting scene cards and if that's the case, add a level two heading.
The check would maintain what the code did before.

Bug @jeanawhou has tested the change and isn't sure what a level 2 heading looks like.

I don't want to hold up the PR if this isn't an important detail.
I'm also struggling to justify why it should remain as it was because I can't provide a good test case to see the change.

A possible test might be to check out staging and export with scene cards turned off and then switch to this branch and export again (with scene cards off) to see the difference.
WDYT?

Reply:
I discussed and made the change with @jeanawhou.